lib/ruby_smb/dcerpc/encrypting_file_system/efs_rpc_open_file_raw_request.rb

```diff
@@ -0,0 +1,21 @@
+module RubySMB
+  module Dcerpc
+    module EncryptingFileSystem
+
+      # [3.1.4.2.1 EfsRpcOpenFileRaw (Opnum 0)](https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-efsr/ccc4fb75-1c86-41d7-bbc4-b278ec13bfb8)
+      class EfsRpcOpenFileRawRequest < BinData::Record
+        attr_reader :opnum
+
+        endian :little
+
+        ndr_conf_var_wide_stringz :file_name
+        ndr_uint32                :flags
```

Comment:
As defined in the specs, would it be possible to add the related constants for these flags?
```
CREATE_FOR_IMPORT 0x00000001
CREATE_FOR_DIR 0x00000002
OVERWRITE_HIDDEN 0x00000004
EFS_DROP_ALTERNATE_STREAMS  0x00000010
```